test/Microsoft.AspNetCore.SignalR.Client.Tests/HubConnectionTests.cs

```diff
@@ -120,23 +120,6 @@ public class HubConnectionTests
             Assert.Same(exception, thrown);
         }
 
-        [Fact]
-        public async Task DoesNotThrowWhenClientMethodCalledButNoInvocationHandlerHasBeenSetUp()
```

Comment:
This test should not have been removed.

Reply:
Replaced this test with a functional test  https://github.com/aspnet/SignalR/blob/mikaelm12/ClientAPIPart2/test/Microsoft.AspNetCore.SignalR.Client.FunctionalTests/HubConnectionTests.cs#L169